portal/conf.py

```diff
@@ -80,7 +81,7 @@
 # Disable Sidebars on special pages
 html_sidebars = {
     'index': [],
-    'gallery': [],
+    'resource-gallery': [],
 }
```

Comment:
```suggestion
    'resource-gallery': [],
    'cookbook-gallery': [],
}
```

Currently the sidebar appears on the cookbook gallery, inconsistent with other pages. I will have time to give this a more thorough review probably later today or tomorrow morning!

Reply:
Thanks Drew!